packages/moleculer-db/src/index.js

```diff
@@ -427,13 +427,13 @@ module.exports = {
 		 * @param {Boolean?} decoding - Need to decode IDs.
 		 * @returns {Object|Array<Object>} Found entity(ies).
 		 */
-		getById(id, decoding) {
+		getById(id, populate, decoding) {
```

Comment:
It's a breaking change, `getById` is an exposed method.

Reply:
Yes, it is. But `populate` for "get" method doesn't work now https://moleculer.services/docs/0.13/moleculer-db.html#get

Reply:
Here is a test which covers that populate works with the "get": https://github.com/moleculerjs/moleculer-db/blob/master/packages/moleculer-db/test/integration/populates.spec.js#L117

Populate implemented in the `transformDocuments` and not in the adapter.